lib/battle/CBattleInfoCallback.cpp

```
@@ -1408,7 +1408,17 @@ AttackableTiles CBattleInfoCallback::getPotentiallyAttackableHexes(
 		attackOriginHex = attacker->occupiedHex(attackOriginHex);
 
 	if (!vstd::contains(defender->getSurroundingHexes(defenderPos), attackOriginHex))
-		throw std::runtime_error("Atempt to attack from invalid position!");
+	{
+		// Fixed in PR #6016 but left as a graceful error log instead of a runtime exception that crashes
+		logGlobal->error(
+			"Attempt to attack from invalid position! defenderPos (%d, %d) attackOriginHex (%d, %d)",
+			defenderPos.getX(),
+			defenderPos.getY(),
+			attackOriginHex.getX(),
+			attackOriginHex.getY()
+		);
+		return AttackableTiles();
+	}
```

Personally crash is way better than silent error, since it often hides problems

---

Ok, removed.